pages/404.tsx

```diff
@@ -0,0 +1,23 @@
+import SignDirectionIcon from 'mdi-react/SignDirectionIcon'
+import * as React from 'react'
+import Layout from '../components/Layout'
+
+export class NotFoundPage extends React.Component<any, {}> {
+    public render(): JSX.Element {
+        return (
+            <Layout location={this.props.location} className="bg-white">
+                <div className="container center error-page text-dark">
+                    <div className="error-page__circle rounded-circle">
```

Comment:
Suggestion: since we are not using `error-page__circle`, `rounded-circle`, `error-page__icon`, we could remove these  classes.